scripts/openmic_split.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+'''OpenMIC train-test partitioning'''
+
+import argparse
+import sys
+
+from tqdm import tqdm
+import pandas as pd
+from sklearn.model_selection import StratifiedShuffleSplit
+
+
+def process_args(args):
+    '''Parse arguments from the command line'''
+
+    parser = argparse.ArgumentParser(description='Split OpenMIC-2018 data into train and test')
+
+    parser.add_argument('metadata',
+                        help='Path to metadata.csv',
+                        type=str)
+
+    parser.add_argument('labels',
+                        help='Path to sparse-labels.csv',
+                        type=str)
+
+    parser.add_argument('-s', '--seed', dest='seed',
+                        default=20180821,
+                        help='Random seed',
+                        type=int)
+
+    parser.add_argument('-n', '--num-splits', dest='num_splits',
+                        default=1,
+                        help='Number of splits to generate',
+                        type=int)
+
+    parser.add_argument('-r', '--split-ratio', dest='ratio',
+                        default=0.75,
+                        help='Fraction of data for training',
+                        type=float)
+
+    return vars(parser.parse_args(args))
+
+
+def load_label_matrix(metadata_file, label_file):
```

Comment:
I would propose moving this function and `make_partitions` into the `openmic` package for easier testing, and on the off chance that we want / need to call this functionality in the notebook tutorials?